apps/prairielearn/elements/pl-variable-output/pl-variable-output.py

```diff
@@ -140,6 +140,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             if np.isscalar(var_data):
                 prefix = ""
                 suffix = ""
+                assert not isinstance(var_data, memoryview)
```

Comment:
Nit, I'd expect to see an assertion be the first thing here in this `if` clause.

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/11158/commits/f4394e5d26dc87a2be97aa2e62bfbb0597418843